docs/reference/overview/visio-javascript-reference-overview.md

```diff
@@ -145,11 +145,11 @@ document.write("<textarea id='ResultOutput' style='width:350px;height:60px'> </t
 document.write("<div id='iframeHost' />");
 
 let session; // Global variable to store the session and pass it afterwards in Visio.run()
-var textArea;
+let textArea;
 // Loads the Visio application and Initializes communication between developer frame and Visio online frame
 function initEmbeddedFrame() {
     textArea = document.getElementById('ResultOutput');
-    var url = document.getElementById('fileUrl').value;
+    let url = document.getElementById('fileUrl').value;
```

Comment:
Does `url` get written to again? Is that why this is a `let` instead of a `const`?

Reply:
Yes, it gets reassigned new values in lines 157-160.

Reply:
ok